src/aics-image-viewer/components/AxisClipSliders/index.tsx

```diff
@@ -128,11 +135,12 @@ export default class AxisClipSliders extends React.Component<AxisClipSlidersProp
 
   createSlider(axis: string, twoD: boolean) {
     const { playing, sliders } = this.state;
+    const numSlices = this.props.numSlices[axis];
     const sliderVals = sliders[axis];
-    const range = { min: 0, max: this.props.numSlices[axis] };
+    const range = { min: 0, max: numSlices - 1 };
 
     return (
-      <div key={axis} className={`slider-row slider-${axis}`}>
+      <div key={axis + numSlices} className={`slider-row slider-${axis}`}>
```

Comment:
I see- it's still possible for the number of z slices to be the same for full field and single cell (less likely for x and y), so you might want to use some even more unique way to generate the key.

Reply:
That case shouldn't be a problem, since the slider range doesn't need to update, and `componentDidUpdate` should handle resetting the handle positions.